data-avro/src/test/java/com/linkedin/data/avro/TestDataTranslator.java

```diff
@@ -50,6 +52,8 @@
 
 public class TestDataTranslator
 {
+  private static final Logger LOG = LoggerFactory.getLogger(TestDataTranslator.class);
```

Comment:
You've said this is only for debug, but it's not actually used in the code you intend to commit. You should delete it.